domain/src/main/java/org/jboss/set/mjolnir/archive/domain/UnsubscribedUsersFromOrgs.java

```diff
@@ -0,0 +1,79 @@
+package org.jboss.set.mjolnir.archive.domain;
+
+import org.hibernate.annotations.CreationTimestamp;
+import org.jboss.set.mjolnir.archive.domain.UserRemoval;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+import java.sql.Timestamp;
+
+@Entity
+@Table(name = "unsubscribed_users_from_orgs")
+public class UnsubscribedUsersFromOrgs {
```

Comment:
Maybe rename to `UnsubscribedUserFromOrg` (singular user and singular org), to make it clear that each instance represents an unsubscription of one particular user from one particular org.

It depends on a convention agreed in each project, but it's common that database table names use plural (`users`) but entity classes use singular (`User`). So keep table names in plural as they are (`unsubscribed_users_from_orgs`);